templates/tweet_eval/emoji/templates.yaml

```diff
@@ -0,0 +1,36 @@
+dataset: tweet_eval
+subset: emoji
+templates:
+  8c794abe-5364-430f-aa1e-eb3501443cec: !Template
+    id: 8c794abe-5364-430f-aa1e-eb3501443cec
+    jinja: "{% set emo = [\n  \"\u2764\",\n  \"\U0001F60D\",\n  \"\U0001F602\",\n\
+      \  \"\U0001F495\",\n  \"\U0001F525\",\n  \"\U0001F60A\",\n  \"\U0001F60E\",\n\
+      \  \"\u2728\",\n  \"\U0001F499\",\n  \"\U0001F618\",\n  \"\U0001F4F7\",\n  \"\
+      \U0001F1FA\U0001F1F8\",\n  \"\u2600\",\n  \"\U0001F49C\",\n  \"\U0001F609\"\
+      ,\n  \"\U0001F4AF\",\n  \"\U0001F601\",\n  \"\U0001F384\",\n  \"\U0001F4F8\"\
+      ,\n  \"\U0001F61C\"] %}\n\nWhich emoji among {{emo}} best describes the sentiment\
```

Comment:
{{emo}} is not compiling, you need to join the list: {{emo | join(", ")}}.
same for `li` and subset sentiment.

Reply:
On my side emo does compile.
Anyway I have added joined emo now